frontend/packages/console-shared/src/hooks/useCanClusterUpgrade.ts

```diff
@@ -1,21 +1,20 @@
 import { useAccessReviewAllowed } from '@console/dynamic-plugin-sdk';
 import { ClusterVersionModel } from '@console/internal/models';
-import { ClusterVersionKind, hasAvailableUpdates } from '@console/internal/module/k8s';
 
 export const isClusterExternallyManaged = (): boolean => {
   return window.SERVER_FLAGS.controlPlaneTopology === 'External';
 };
 
-export const useCanClusterUpgrade = (clusterVersion: ClusterVersionKind): boolean => {
+export const useCanClusterUpgrade = (): boolean => {
   const hasPermissionsToUpdate = useAccessReviewAllowed({
     group: ClusterVersionModel.apiGroup,
     resource: ClusterVersionModel.plural,
     verb: 'patch',
     name: 'version',
   });
   const notExternallyManaged = !isClusterExternallyManaged();
-  const bandingNotDedicated = window.SERVER_FLAGS.branding !== 'dedicated';
-  const canPerformUpgrade = hasPermissionsToUpdate && bandingNotDedicated && notExternallyManaged;
+  const brandingNotDedicated = window.SERVER_FLAGS.branding !== 'dedicated';
+  const canPerformUpgrade = hasPermissionsToUpdate && brandingNotDedicated && notExternallyManaged;
 
-  return hasAvailableUpdates(clusterVersion) && canPerformUpgrade;
+  return canPerformUpgrade;
```

Comment:
@andrewballantyne, see https://github.com/openshift/console/pull/11363#issuecomment-1103234458.  Because this hook is also used to hide the channel link, the `hasAvailableUpdates` check has to be done separately for the case when the channel isn't set as there are no updates available when a channel isn't set.

Reply:
cc:  @jhadvig, @zherman0 

Note this update no longer requires the `ClusterVersion`, which greatly simplifies the use of the hook.